lib/rouge/regex_lexer.rb

```diff
@@ -289,11 +290,11 @@ def stream_tokens(str, &b)
     def step(state, stream)
       state.rules.each do |rule|
         if rule.is_a?(State)
-          puts "  entering mixin #{rule.name}" if @debug
+          puts "  entering: mixin :#{rule.name}" if @debug
           return true if step(rule, stream)
-          puts "  exiting  mixin #{rule.name}" if @debug
+          puts "  exiting:  mixin :#{rule.name}" if @debug
```

Comment:
I realise this is a behaviour from the prior code but do you think it makes sense to ensure `mixin` is indented to the same level for both the `entering` and `exiting` statements?

When viewed in isolation, I see the appeal but, when looking at your example output, this consistency seems barely noticeable (and is arguably inconsistent with the indentation with the other lines).

Reply:
Oh, and just to be clear, by 'indented' here I mean the extra space _after_ `exiting`.

Reply:
Hmm. I see where you're getting at..
I'll push a fix.

Reply:
Thanks! Looks good :)